### DISCUSSION TITLE
Rtc versus pfc8563

### DISCUSSION THREAD

**Discussion by gitmun** [micropython/micropython#11579] [RP2040 / Pico]
Hello, I have a question and the truth is that I don't know what else to try... I have modified a project that had a ds3231 replacing it with a pfc8563, I only needed the time and the day, but what is my surprise when it is running from the ide (thonny) it works perfectly, the time is correct and the day, but when you leave it independent, the default time and day of the internal RTC appear... is there any way to solve this? the case that asking the rtc is the correct date

**Comment by robert-hh** [comment]
On startup Thonny updates the date and time from the PC's clock.

**Reply by icelon** [reply]
yes , sure it is like that , but once the day time was updated ... in the pcf8563 , powered by batteries ... I had to save the programming ... which is correct but when updating the rtc of the rp2 passing it a tuple ... lose both ....

**Reply by icelon** [reply]
e6609cb2d31d3e29  1-01-2021 00:00:02 result for battery , and update from RTC  pcf8563 ...(2023, 5, 21, 15, 57, 28, 6, 1) not update rtc of rp2

**Comment by icelon** [comment]
Could it be that it is not updated and the value sent is reset? If so, how could I save the dates and times before connecting it to the batteries?

**Comment by icelon** [comment]
I found the error when passing the tuple I ate the ()
